README.md

```diff
@@ -31,7 +38,9 @@ If you have any questions, comments, insights, praises, or doubts about these do
 
 # Limitations
 
-Current version is `pre-alpha` which means the happy path is working on Linux on the Golem test network. 
+Current version is `pre-alpha` which means the happy path is working on Ubuntu on the Golem test network.
+We have tested Ray on Golem on Ubuntu and on WSL. It should work on other Linux distributuions, might work on MacOS and shouldn't on bare Windows.
+ 
 We use this version to show the direction and get feedback.
 
 There is one Ray on Golem image. It contains `ray 2.3.1` and `python 3.10.12`.
```

Comment:
I wouldn't mention patch versions, especially that I believe it's not specifically 3.10 _.12_ right now... 

```suggestion
There is one Ray on Golem image. It contains `ray 2.3` and `python 3.10`.
```